7.11 Capstone 1 - Analytic Report and Research Proposal.ipynb

```diff
@@ -377,29 +336,30 @@
     "\n",
     "dfs = [df2014,df2015,df2016,df2017]\n",
     "deaths_total = df2014['DEATHS'] + df2015['DEATHS'] + df2016['DEATHS'] + df2017['DEATHS']\n",
-    "\n",
     "dftotals['TOTAL DEATHS'] = deaths_total\n",
-    "dftotals['Total Background Checks'] = st_totals\n",
-    "print(dftotals)\n",
+    "# print(dftotals)\n",
+    "# print(st_totals)\n",
+    "dftotals['Total Background Checks'] = st_totals['totals']\n",
+    "# print(dftotals)\n",
     "\n",
-    "#plt.figure(figsize=(12,7))\n",
-    "#ax1 = sb.stripplot(x=\"STATE\", y=\"TOTAL DEATHS\", data=dftotals, jitter=0.05, color='purple')\n",
-    "#ax2 = sb.stripplot(x='state', y='totals', data=st_totals, color='green')\n",
-    "#ax2 = ax1.twinx()\n",
+    "plt.figure(figsize=(12,7))\n",
+    "ax1 = sb.stripplot(x=\"STATE\", y=\"TOTAL DEATHS\", data=dftotals, color='purple')\n",
```

Comment:
jitters wasnt loading so i removed it, maybe my versions old so you can re add...